tests_config/run_spec.js

```diff
@@ -25,6 +25,7 @@ const unstableTests = new Map(
     "class_comment/comments.js",
     ["comments/dangling_array.js", (options) => options.semi === false],
     ["comments/jsx.js", (options) => options.semi === false],
+    "comments/binary-expressions-single-comments.js",
```

Comment:
I hate this list growing, but not related to this PR, good job.

Reply:
I agree. I have ideas.

1. Unstable test files have `// Flag: unstable test` comment on the top.
1. Make `tests/unstable` directory and move unstable test files into it.
1. Move this list to another file. (e.g. `unstable_tests.json`) This is easiest to implement.

Do you have any ideas?

Reply:
It's okay to keep this list here, easy to trace.

we need kill them all, and we will.